<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Student Management</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>List of students</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget_students">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Age</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Intro</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_studentdetails">
      <property name="title">
       <string>Student Details</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_id"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_intro">
         <property name="text">
          <string>Intro</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QLabel" name="label_code">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" rowspan="2">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_code"/>
       </item>
       <item row="6" column="0" rowspan="2">
        <widget class="QLabel" name="label_age">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_name"/>
       </item>
       <item row="7" column="2" rowspan="3">
        <widget class="QLineEdit" name="lineEdit_intro"/>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="pushButton_avatar">
         <property name="text">
          <string>Avatar</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QPushButton" name="pushButton_removeavatar">
         <property name="text">
          <string>Remove Avatar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="7" colspan="2">
        <widget class="QLabel" name="label_image">
         <property name="maximumSize">
          <size>
           <width>164</width>
           <height>98</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/DELL/Pictures/Saved Pictures/4a4c91b1d4a2cd19cf62e22ab0a94bd9.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEdit_age"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_action">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_insert">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
